server/routes/snippets.js

```diff
@@ -11,21 +11,26 @@ var authChecker = function(req, res, next) {
   }
 };
 
-/* GET snippets listing. */
+/* GET snippets from specified month */
 router.get('/', function (req, res) {
-  var perPage = req.query.results;
-  var page = req.query.offset;
+  /* January is 0, December is 11 */
+  var options = { where: {}, order: [] };
+  var now = new Date(Date.now());
+  var month = parseInt(req.query.month);
+  var year = parseInt(req.query.year);
+  var first = new Date(year, month, 1);
+  var last = new Date(year, month+1, 0);
+  options.where = { createdAt: { $gte: first, $lte: last } };
+  
+  options.order = (now.getFullYear() == year && now.getMonth() == month) ? [['createdAt', 'DESC']] : [ ['avg', 'DESC'], ['createdAt', 'DESC'] ];
```

Comment:
I think we've agreed that snippets index list will be simple ordered by creation date and there will be other button/select box to get snippets by `avg`. So index action should take two sets of parameters one for pagination and second for sort order. Am I right @blimusiek?